README.md

```diff
@@ -346,6 +346,21 @@ To stop sharing existing factors created with **App Groups**, uncheck the **App
 
 > This will restrict access to the factors and will not affect the main application in which the data was initially created.
 
+---
+
+## Allow Factors migration
+
+By default, all factors are stored using the accessibility attribute [kSecAttrAccessible](https://developer.apple.com/documentation/security/ksecattraccessible) with a value of [kSecAttrAccessibleAlwaysThisDeviceOnly](https://developer.apple.com/documentation/security/ksecattraccessiblealwaysthisdeviceonly). This means that the item stored with this value can be restored to the same device that created a backup, but it isn't migrated when restoring another device's backup data.
```

Comment:
Should the value be [kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly](https://developer.apple.com/documentation/security/ksecattraccessibleafterfirstunlockthisdeviceonly) (value and link)?